src/test/scala/dpla/ingestion3/model/JsonlStringTest.scala

```diff
@@ -0,0 +1,17 @@
+package dpla.ingestion3.model
+
+
+import org.scalatest.FlatSpec
+import dpla.ingestion3.data.EnrichedRecordFixture
+import org.json4s._
+import org.json4s.jackson.JsonMethods._
+
+class JsonlStringTest extends FlatSpec {
+
+  "jsonlRecord" should "print a valid JSON string" in {
```

Comment:
Think there should be more tests here. Parse the json and find out if the properties came through?